ci/workflows/main-cron.yml

```diff
@@ -280,7 +280,8 @@ steps:
     plugins:
       - docker-compose#v5.5.0: *docker-compose
       - ./ci/plugins/upload-failure-logs
-    timeout_in_minutes: 90
+    timeout_in_minutes: 40
+    parallelism: 4
```

Comment:
So we added parallel ability to `deterministic-it-test.sh`, and enabled it only for `deterministic-it-test.sh scale::`, right?

Reply:
Exactly